.circleci/config.yml

```diff
@@ -142,6 +142,9 @@ jobs:
       - install_chrome
       - attach_workspace:
           at: /home/circleci/
+      - run:
+          name: Setup python by installing protobuf and enum
+          command: pip install enum34==1.1.10 protobuf==3.13.0
```

Comment:
Question: Why do we have to install these separately instead of as part of the setup scripts (e.g. `install_third_party.py`)?

Reply:
The installation does not persist to when we commit to the directory since we only persist the oppia home directory.

Reply:
We persist `oppia_tools`, `oppia/third_party`, and `oppia/node_modules`. Does this not cover what `install_third_party.py` installs?

https://github.com/oppia/oppia/blob/bd8f6376bb8fb837bd48332ba4f5810d59e39142/.circleci/config.yml#L127-L132

Reply:
Yes, this is not a local installation, or else it would have been listed in install_third_party_libs. This is installed to the system.

Reply:
Okay, that makes sense